.htaccess.example

```diff
@@ -106,5 +107,5 @@
 
 </IfModule>
 
-php_value upload_max_filesize 50M
-php_value post_max_size 50M
+php_value upload_max_filesize 500M
```

Comment:
This should still be `50M` (a more sensible value). Since you seem a bit confused, let me try to clarify: 

The idea is to make `.htaccess` opt-in and fully customizable. For this to happen, it should NOT be version-controlled. Instead, a (version-controlled) sample file (`.htaccess.example`) should exist and contain the default, sensible configuration. If `.htaccess` doesn't already exist, the sample file will be copied to create one. Since, again, `.htaccess` isn't version-controlled, the next update will NOT overwrite it, ensuring all customizations are kept. This is a common practice.